### ISSUE TITLE
Mocking SendMailV31Func of the HTTPClientMock

### ISSUE THREAD

**Issue by bilias23** [mailjet/mailjet-apiv3-go#55]
Hello and thanks for providing the mocks, they are really helpful.

One minor suggestion I'd have is to change the return of the default `SendMailV31Func` in the `NewhttpClientMock`. 
Returning `nil, nil` is guaranteed to cause a panic, which will likely need some debugging for each developer stumbling upon this.
While a panic is a good way to get someone's attention, maybe it would be better to return by default an information error something along the lines of "mock send mail function not implemented yet", or "a mock send mail function needs to be implemented on your side"

In short I 'd suggest that ln 40 in `http_client_mock.go` would be changed to something like
`return nil, errors.New("mock send mail function not implemented yet")`

Thanks!

**Comment by todorDim** [comment]
Hey @bilias23,

Thank you for the suggestion. We'll take it into consideration when updating the wrapper.

Regards,
T.

**Comment by gertcuykens** [comment]
I would vote for `return something, nil` instead to be consistent with the other cals that also return some mock data?

Also wouldn't it be nice to just be able to replace
```
var MC = mj.NewMailjetClient("...", "...")
```
with
```
var MC = mj.NewhttpClientMock(true)
```
Currently you can't because of the `SendMailV31` signature?
